src/Microsoft.Health.Fhir.R4.Core.UnitTests/Microsoft.Health.Fhir.R4.Core.UnitTests.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Extensions.Xunit\Microsoft.Health.Extensions.Xunit.csproj" />
+    <ProjectReference Include="..\Microsoft.Health.Fhir.Azure\Microsoft.Health.Fhir.Azure.csproj" />
```

Comment:
>Azure [](start = 84, length = 5)

I find strange what Azure package is creeping inside core tests.
If I correctly understand it's because of `ContainerRegistryBasicTokenProvider` class.
Can it be mocked instead?

Reply:
Yes. I have add a mock for that library